### PR TITLE
Unit Test CommentController

### PR DESCRIPTION
**Pull request by hungsondo**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
- Unit Test CommentController

## Notes Impacted Areas:
- N/A.

## Attachments:
- image

### CODE REVIEW

**Review by longnv-99 — APPROVED** [review]

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -10,26 +11,13 @@
...
-     * @return \Illuminate\Http\Response
-     */
-    public function create()
+    public function __construct(CommentRepositoryInterface $commentRepo)
     {
-        //
+        $this->commentRepo = $commentRepo;
     }
-
+
```
Comment (truongcr7) [inline app/Http/Controllers/CommentController.php:20]: bỏ tab này

File: app/Repositories/Comment/CommentRepository.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{
+    public function getModel()
+    {
+        return Comment::class;
+    }
+
+    public function hideComment($id)
+    {
+        return $this->update($id, [
+            'display' => config('app.non-display')
```
Comment (truongcr7) [inline app/Repositories/Comment/CommentRepository.php:18]: dấu phẩy ở cuối

File: app/Repositories/Comment/CommentRepository.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        ]);
+    }
+
+    public function showComment($id)
+    {
+        return $this->update($id, [
+            'display' => config('app.display')
```
Comment (truongcr7) [inline app/Repositories/Comment/CommentRepository.php:25]: tt

File: tests/Unit/Http/Controllers/CommentTest.php [hunk 4]
```diff
@@ -0,0 +1,229 @@
+<?php
```
Comment (truongcr7) [inline tests/Unit/Http/Controllers/CommentTest.php:1]: tên file đặt là CommentControllerTest thì đúng hơn

**Review by truongcr7 — APPROVED** [review]

**Comment by hungsondo** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by hungsondo** [comment]
ready

**Pull request merged**